packages/react-bootstrap-table2/src/props-resolver/index.js

```diff
@@ -40,4 +40,78 @@ export default ExtendBase =>
         ...cellEditInfo
       };
     }
+
+    /**
+     * props resolver for cell selection
+     * @param {Object} options - addtional options like callback for cell selection
+     *
+     * @returns {Object} result
+     * @returns {String} result.mode - input type of row selection or disabled.
+     */
+    resolveCellSelectionProps(options) {
+      const { selectRow } = this.props;
+      const { ROW_SELECT_SINGLE, ROW_SELECT_MULTIPLE, ROW_SELECT_DISABLED } = Const;
+
+      if (_.isDefined(selectRow)) {
+        let { mode } = selectRow;
+
+        if (!mode || (mode !== ROW_SELECT_SINGLE && mode !== ROW_SELECT_MULTIPLE)) {
+          mode = Const.ROW_SELECT_MULTIPLE;
```

Comment:
Yep. So far I allow user to set mode to `ROW_SELECT_SINGLE` or `ROW_SELECT_MULTIPLE ` only. Other values or empty are prohibited and it will back to default `ROW_SELECT_MULTIPLE `.